### PR TITLE
Update PPP and UG

### LINKED ISSUE #320: Show an example for setting unhealthy goal
image

I think it will be easier for user if we have an example for setting `unhealthy` goals

### LINKED ISSUE #324: Diet Goal type did not change from Unhealthy to Healthy with `unhealthy` flag
image

From UG: 
```
* unhealthy: This determines if you are trying to get more of this nutrient or less of it. 
This flag is used to change goals that are set as unhealthy previously.
```

### PR DESCRIPTION
**Pull request by yicheng-toh**
Explain feature and enhancement in PPP and add example output to diet goal user guide.

Fixes #320
Fixes #324

### CODE REVIEW

**Review by skylee03 — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -417,23 +416,50 @@ You can set multiple nutrients goals at once with the `set-diet-goal` command.
...
+  _____________________________________________________________
+  These are your goal(s):
+  
+      1. [HEALTHY]  WEEKLY calories intake progress: (0/500)
+  
+      2. [HEALTHY]  WEEKLY fats intake progress: (0/600)
+  
+  Now you have 2 diet goal(s).
+  _____________________________________________________________
+```
```
Comment (nihalzp) [inline docs/UserGuide.md:462]: We have added picture in the other part of the UG. Should we be consistent here as well?

File: docs/team/yicheng-toh.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+...
+### Contributions to team-based tasks
+
+* Kept track of deadlines for v1.0 and v2.0.
+* Assisted in sorting and assigning of post PE dry run issues.
+* Suggested the use of interface for find function and abstract class for goals. 
+This is only implemented due to  [skylee03 (Ming-Tian)](./skylee03.md)'s outstanding effort in convincing the team.
+* Examples of PR reviewed: 
+  * [PR for editing activities](https://github.com/AY2324S1-CS2113-T17-1/tp/pull/59#discussion_r1362968136)
+* Provided full scale testing for diet functionalities.
```
Comment (nihalzp) [inline docs/team/yicheng-toh.md:91]:
```suggestion
* Provided full scale testing for diet goal functionalities.
```

**Review by nihalzp — CHANGES_REQUESTED** [review]
Additionally, we only have usage pictures for `list` commands; it seems it might look inconsistent if we provide example usage for all commands of `diet-goal`

File: docs/team/yicheng-toh.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+...
+Which sections did you contribute to the DG? 
+Which UML diagrams did you add/updated?
+...
+### Contributions to team-based tasks
+
+* Kept track of deadlines for v1.0 and v2.0.
+* Assisted in sorting and assigning of post PE dry run issues.
+* Suggested the use of interface for find function and abstract class for goals. 
+This is only implemented due to  [skylee03 (Ming-Tian)](./skylee03.md)'s outstanding effort in convincing the team.
```
Comment (skylee03) [inline docs/team/yicheng-toh.md:88]:
```suggestion
This is only implemented due to  [skylee03 (Ming-Tian)](./skylee03.html)'s outstanding effort in convincing the team.
```

**Pull request merged**